pkg/config/attributed_string_slice.go

```diff
@@ -0,0 +1,84 @@
+package config
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/BurntSushi/toml"
+)
+
+// attributedStringSlice allows for extending a TOML string array with custom
+// attributes that control how the array is marshaled into a Go string.
+//
+// Specifically, an attributedStringSlice can be configured to avoid it being
+// overridden by a subsequent unmarshal sequence.  When the `append` attribute
+// is specified, the array will be appended instead (e.g., `array=["9",
+// {append=true}]`).
+type attributedStringSlice struct { // A "mixed-type array" in TOML.
+	// Note that the fields below _must_ be exported.  Otherwise the TOML
+	// encoder would fail during type reflection.
+	Slice      []string
```

Comment:
Maybe a nitpick but can we rename this to `Values`? Seeing all these Env.Slice looks of compared to something like Env.Values

Reply:
I like it, thanks.